jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -282,13 +277,21 @@ private static float getBuildTimeFromVerifierLog(Verifier verifier) throws IOExc
     return -1;
   }
 
-  private static void assertCreationTimeEpoch(String imageReference)
+  private static void assertCreationTimeEpochPlusOne(String imageReference)
       throws IOException, InterruptedException {
     Assert.assertEquals(
-        "1970-01-01T00:00:00Z",
+        "1970-01-01T00:00:01Z",
         new Command("docker", "inspect", "-f", "{{.Created}}", imageReference).run().trim());
   }
 
+  private static void assertCreationTimeIsAfter(Instant before, String imageReference)
+      throws IOException, InterruptedException {
+    String inspect =
+        new Command("docker", "inspect", "-f", "{{.Created}}", imageReference).run().trim();
+    Instant after = Instant.parse(inspect);
+    Assert.assertTrue(after.isAfter(before) || after.equals(before));
```

Comment:
Probably it's safe to remove `equals(before)`? It is desirable because otherwise the assertion will also succeed for a fixed timestamp. This should be for explicitly testing USE_CURRENT_TIMESTAMP is working.